ks-svarut/src/main/java/no/difi/meldingsutveksling/ks/mapping/HandlerCollection.java

```diff
@@ -0,0 +1,23 @@
+package no.difi.meldingsutveksling.ks.mapping;
+
+import no.difi.meldingsutveksling.ks.Forsendelse;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Main mapper entrypoint for EDUCore (EDU payload)
+ *
+ * Used to map EDUCore to Forsendelse (SvarUt payload)
+ */
+public class HandlerCollection implements Handler<Forsendelse.Builder> {
+    List<Handler<Forsendelse.Builder>> handlers = new ArrayList<>();
+
+    @Override
+    public Forsendelse.Builder map(Forsendelse.Builder builder) {
+        for (Handler<Forsendelse.Builder> handler : handlers) {
+            builder = handler.map(builder);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Introduce a new variable instead of reusing the parameter "builder". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1226)